assets/views/head.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
This file isn't using relative paths.

Reply:
My mistake.
That was a test that I'll delete